File: Layer_datas/Services/Objects/activities/Session.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Layer_datas.Services.Objects.activities
{
    public class Session
    {
        public int id { get; set; }
        public int day { get; set; }
        public TimeSpan start { get; set; }
        public TimeSpan end { get; set; }
        public int nbSlots { get; set; }

        public Place place { get; set; }
        public Section section { get; set; }

    }
}

File: Swiman/__scripts__/modules/api/models/Session.ts


    
        import { Place } from "./Place";
    
        import { Section } from "./Section";
    

    import * as assign from "object-assign";

    

    


    class Session {
        

        
        // ID
        public id: number = 0;
        // DAY
        public day: number = 0;
        // START
        public start: string = undefined;
        // END
        public end: string = undefined;
        // NBSLOTS
        public nbSlots: number = 0;
        // PLACE
        public place: Place = undefined;
        // SECTION
        public section: Section = undefined;

        constructor(init?: Partial<Session>){
            assign(this, init);
        }
        
    }


export {  Session }



